.vscode/settings.json

```diff
@@ -9,7 +9,7 @@
         "dlvLoadConfig": {
             "followPointers": true,
             "maxVariableRecurse": 1,
-            "maxStringLen": 255,
+            "maxStringLen": 8192,
```

Comment:
Oh such a good call - I should have done this long ago and have no idea why delve defaults to 255 then `...`